lighthouse-extension/app/src/lighthouse-background.js

```diff
@@ -25,61 +25,21 @@ const log = require('../../../lighthouse-core/lib/log');
 
 const ReportGenerator = require('../../../lighthouse-core/report/report-generator');
 
-const STORAGE_KEY = 'lighthouse_audits';
+const STORAGE_KEY = 'lighthouse_v2';
```

Comment:
You probably have to remove ligthouse_audits from the storage so it doesn't stay alive (`onInstalled`)